README.md

```diff
@@ -1220,6 +1220,8 @@ With`extra_settings`, you can pass multiple custom settings via the `awx-operato
 | -------------- | -------------- | ------- |
 | extra_settings | Extra settings | ''      |
 
+**Note:** Parameters configured in `extra_settings` cannot change after deployed awx.
```

Comment:
I don't believe this is true... @rooftopcellist @TheRealHaoLiu is there a bug here?

Reply:
i dont think this is true, it will cause a rolling restart so that the setting takes effect i think

Reply:
Can we add a bit more detail to the doc note?

```suggestion
**Note:** Parameters configured in `extra_settings` are set as read-only settings in AWX.  As a result, they cannot be changed in the UI after the fact.  
```